src/test/java/com/bmdsoftware/pacs/dicoogle/ipfs/TestIPFS.java

```diff
@@ -22,68 +22,64 @@
 import io.ipfs.api.MerkleNode;
 import io.ipfs.api.NamedStreamable;
 import io.ipfs.multihash.Multihash;
-import org.junit.Ignore;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Map;
+import java.util.UUID;
 
-public class TestIPFS {
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
+/**
+ * @author Rui Lebre - <ruilebre@ua.pt>
+ */
+public class TestIPFS {
+    private static IPFS ipfs;
+    private static String testFileContent = "test test test";
 
+    @BeforeClass
+    public static void setup() {
+        ipfs = new IPFS("/ip4/127.0.0.1/tcp/5001");
+    }
 
     @Test
-    @Ignore // needs isolation
-    public void ipfs() throws IOException {
-
-
-        IPFS ipfs = new IPFS("/ip4/127.0.0.1/tcp/5001");
-        //IPFS ipfs = new IPFS("/ip4/104.236.179.241/tcp/4001");
-
+    public void testFileOperations() throws IOException {
+        NamedStreamable.ByteArrayWrapper file = new NamedStreamable.ByteArrayWrapper(UUID.randomUUID().toString(), testFileContent.getBytes(StandardCharsets.UTF_8));
```

Comment:
This tests was initiail wrote only to do some experimental actually. And it is fine as it is.
As suggestions, perhaps we can do a more encapsulated tests to verify the right behaviour of `at` and `store` method of `IPFSStore`